zkevm-circuits/src/state_circuit/state.rs

```diff
@@ -68,14 +68,20 @@ Example bus mapping:
 |    3   |    1    |       49       |  32   |             |            |  0   |
 */
 
+const EMPTY_TAG: usize = 0;
```

Comment:
I have done a brief review on the changes in this file.  I see that there are many things that are going to change to update this according to the latest specs, hence why I only reviewed it briefly.